libs/wire-api/test/unit/Test/Wire/API/MLS.hs

```diff
@@ -214,6 +216,49 @@ testRemoveProposalMessageSignature = withSystemTempDirectory "mls" $ \tmp -> do
       )
       Nothing
 
+testParseGroupInfoBundle :: IO ()
+testParseGroupInfoBundle = withSystemTempDirectory "mls" $ \tmp -> do
+  qcid <- do
+    let c = newClientId 0x3ae58155
+    usr <- flip Qualified (Domain "example.com") <$> (Id <$> UUID.nextRandom)
+    pure (userClientQid usr c)
+  void . liftIO $ spawn (cli qcid tmp ["init", qcid]) Nothing
```

Comment:
Since #2678 was merged, I think we can use that instead, right?

Reply:
Should we extract those into their own internal test lib so we can use them across the codebase? @pcapriotti ?

Reply:
I'm not to keen on doing this. It's not that much code, anyway, and the unit test setup is much simpler anyway, since we only need to extract one particular message, and not keep track of the state of several clients.